src/gen/conditions.rs

```diff
@@ -269,12 +290,13 @@ impl ParseState {
 }
 
 fn parse_spend_conditions(
+    ret: &mut SpendBundleConditions,
```

Comment:
You can make this a method of `SpendBundleConditions`.
```suggestion
impl SpendBundleConditions {
    fn parse_spend_conditions(&mut self, ...
```

Reply:
yes, but I don't see any benefits with doing so